src/Features/Core/Portable/Completion/CommonCompletionItem.cs

```diff
@@ -37,7 +37,8 @@ internal static class CommonCompletionItem
             string filterText = null,
             bool showsWarningIcon = false,
             ImmutableDictionary<string, string> properties = null,
-            ImmutableArray<string> tags = default)
+            ImmutableArray<string> tags = default,
+            string inlineDescription = null)
```

Comment:
this might be an issue for binary compat with our partners.  need to check on that.

Reply:
remember to check this.

Reply:
Yup, it does crash F#

Reply:
Fixed